erpnext/config/integrations.py

```diff
@@ -19,6 +19,17 @@ def get_data():
 				}
 			]
 		},
+		{
+			"label": _("Webhook"),
```

Comment:
This should be removed since this will be visible for all. We are working on new desk and module pages where its fully customizable, so this can be added later on